### PR TITLE
initial commit + portal and auth modules

### PR DESCRIPTION
**Pull request by SlothOfAnarchy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+targetcli-modules
+=================
```
Comment (dasJ) [inline README.md]: Use the other mardown syntax. This one is ugly

File: README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+Modules for interacting with targetcli command line utility.
+
+Requirements
+------------
+`targetcli` has to be installed
+
+License
+-------
+
+GPLv3
```
Comment (dasJ) [inline README.md]: Consider relicensing

File: meta/main.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+galaxy_info:
+  author: Ondrej Faměra, Michel Weitbrecht
+  description: modules for managing targetcli.
```
Comment (dasJ) [inline meta/main.yml]: Capitalize

File: meta/main.yml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  license: GPLv3
+  min_ansible_version: 2.0
+  platforms:
+  - name: EL
```
Comment (dasJ) [inline meta/main.yml]: owl

**Review by dasJ — DISMISSED** [review]
Not willing to read the python part, this is all I can do

**Comment by SlothOfAnarchy** [comment]
@dasJ please rereview

File: library/targetcli_backstore.py [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+                        backstore_type=dict(required=True),
+                        backstore_name=dict(required=True),
+                        options=dict(required=False),
+                        state=dict(default="present", choices=['present', 'absent']),
+                ),
+                supports_check_mode=True
+        )
+
+        state = module.params['state']
+        if state == 'present' and not module.params['options']:
```
Comment (Scriptkiddi) [inline library/targetcli_backstore.py:60]: string compare mit == nimm lieber starte is 'present'

**Review by Scriptkiddi — APPROVED** [review]

**Pull request merged**